docs/development/index.rst

```diff
@@ -13,6 +13,7 @@ Guidelines
    CloudDevGuideline
    PRGuideline
    TicketLifecycle
+   BranchManagement
```

Comment:
I am thinking, do we want to merge PRGuideline and BranchManagement and create a new doc named "Git Management"?

Reply:
yes agreed. was missing the bigger picture here. merged git related docs in commit [b5afa7f](https://github.com/CraftsmenLtd/BloodConnect/pull/111/commits/b5afa7f41064a3d53d68aa226af00a5354eb744e)